package.json

```diff
@@ -12,7 +12,7 @@
     "tsc:noemit": "tsc --noemit",
     "lint": "tslint -p . -c tslint.json -t verbose",
     "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
-    "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.21/api_proxy.yaml --out-dir ./definitions/backend"
+    "generate:api-definitions": "yarn rimraf definitions/backend && mkdir -p definitions/backend && yarn gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v0.0.21/api_proxy.yaml --out-dir ./definitions/backend"
```

Comment:
Are you sure we need this?

Reply:
it can be avoided. anyway, on my laptop I need to launch the 3 commands manually (the command is not recognized)

Reply:
Doesn't `yarn generate:api-definitions` work on your laptop?

Reply:
no it doesn't. the same code work well if I execute the 3 commands manually.

Reply:
@valentinacala I suppose you are on windows?